openpype/hosts/maya/plugins/create/create_render.py

```diff
@@ -220,6 +220,12 @@ def _create_render_settings(self):
         self.data["tilesY"] = 2
         self.data["convertToScanline"] = False
         self.data["useReferencedAovs"] = False
+        self.data["renderSetupIncludeLights"] = (
+            self._project_settings.get(
+                "maya", {}).get(
+                "RenderSettings", {}).get(
+                "enable_all_lights", {})
```

Comment:
```suggestion
                "enable_all_lights", False)
```

This should not default to a dict if `enable_all_lights` isn't found because a `dict` isn't settable on the created node and also is the wrong type?